examples/pwa/pwa.html

```diff
@@ -11,6 +11,33 @@
     if (window.location.pathname.indexOf('/examples') == 0) {
       window.location.assign('/pwa');
     }
+
+    function loadScript(url, callback) {
+        // Adding the script tag to the head as suggested before
+        var head = document.getElementsByTagName('head')[0];
+        var script = document.createElement('script');
+        script.type = 'text/javascript';
+        script.src = url;
+
+        // Then bind the event to the callback function.
+        // There are several events for cross browser compatibility.
+        script.onreadystatechange = callback;
+        script.onload = callback;
```

Comment:
To be honest, this was written as just a proof of concept, and I grabbed this from some stack overflow example. I used this to just console log what was happening to make the flow easier to follow.

The `webcomponents.js` does have an event that we can listen to: https://github.com/webcomponents/webcomponentsjs#webcomponents-loaderjs

Though I remember you said you didn't want to be dependent on one polyfills API, thus I did not add this.